src/sentry/static/sentry/app/views/settings/incidentRules/triggers/actionsPanel/index.tsx

```diff
@@ -54,47 +53,61 @@ type Props = {
   onChange: (triggerIndex: number, triggers: Trigger[], actions: Action[]) => void;
 };
 
-const getPlaceholderForType = (type: ActionType) => {
-  switch (type) {
-    case ActionType.SLACK:
-      return '@username or #channel';
-    case ActionType.MSTEAMS:
-      //no prefixes for msteams
-      return 'username or channel';
-    case ActionType.PAGERDUTY:
-      return 'service';
-    default:
-      throw Error('Not implemented');
-  }
+/**
+ * When a new action is added, all of it's settings should be set to their default values.
+ * @param actionConfig
+ */
+const getCleanAction = (actionConfig): Action => {
+  return {
+    type: actionConfig.type,
+    targetType:
+      actionConfig &&
+      actionConfig.allowedTargetTypes &&
+      actionConfig.allowedTargetTypes.length > 0
+        ? actionConfig.allowedTargetTypes[0]
+        : null,
+    targetIdentifier: actionConfig.integrationId || '',
```

Comment:
I don't know that we want this to default to the `integrationId` for anything other than sentry apps cause won't that just put the integration id in the text box?